staging/src/k8s.io/apimachinery/pkg/runtime/serializer/json/json.go

```diff
@@ -93,6 +95,9 @@ type SerializerOptions struct {
 	// Strict: configures the Serializer to return strictDecodingError's when duplicate fields are present decoding JSON or YAML.
 	// Note that enabling this option is not as performant as the non-strict variant, and should not be used in fast paths.
 	Strict bool
+
+	// DisableCollectionStreaming turns off streaming for collections.
+	StreamingCollections bool
```

Comment:
thanks! i like this better. (need to fix `DisableCollectionStreaming` in the comment in the line above)

Reply:
Done